.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
Yes it was supposed to be included. I agree it makes more sense as a separate PR